app/src/main/java/org/thoughtcrime/securesms/ui/Components.kt

```diff
@@ -126,7 +125,6 @@ fun ItemButtonWithDrawable(
     @StringRes textId: Int,
     @DrawableRes icon: Int,
     colors: ButtonColors = transparentButtonColors(),
-    @StringRes contentDescription: Int = textId,
```

Comment:
Don't you need the contentDescription?
It was meant to be removed from the image but the internal ItemButton still need to be set with something

Reply:
will just put it on the modifier... one sec.

Reply:
You can have it the way you had it,except set to the Imagebutton not the Image

Reply:
`ItemButton` is pretty old. it seems to make more sense to me to add it as a modifier, as `contentDescription` can apply to almost anything, and we don't always want it, so modifier suits that use case better. done.

Reply:
I'm not sure I agree. contentDescription in a Composable should mean the content description for that object.
Having it in the method enforces the responsibility of the caller to fulfil the ax requirement

Reply:
well, `contentDescription` are required by default only on graphical components like `Icon`s. Whereas contentDescriptions for buttons are not required to be supplied as they can be constructed from some combination of `button type, button state, button content` based on screen reader preference.

But anyway... we don't use contentDescriptions properly, we use them for testing, and we only need to put them on buttons so Appium can reference these.